src/implementations/vanilla/components/site-header/site-header.js

```diff
@@ -255,17 +255,29 @@ export class SiteHeader {
     }
 
     // Check total width, and change display if needed
-    this.languageListEu.parentNode.classList.remove(
-      'ecl-site-header__language-content--stack',
-    );
+    if (this.languageListEu) {
```

Comment:
not easy to understand the relation between this selector and the close button, guess that it was not working because a js error thrown by these "lines", and i don't see any error regardless of the number, or the presence of the two lists, so...good as it is! :)